homeassistant/components/lyric/climate.py

```diff
@@ -268,6 +275,16 @@ def max_temp(self) -> float:
             return device.maxHeatSetpoint
         return device.maxCoolSetpoint
 
+    @property
+    def fan_mode(self) -> str | None:
+        """Return current fan mode."""
+        device = self.device
+        return (
```

Comment:
Need to translate "Circulate" from the Resideo API to to FAN_DIFFUSE for HA

Reply:
mapped